### PR TITLE
Updated save_the_date.py

### PR DESCRIPTION
**Pull request by dillwishlist**
Added context definitions to get_save_the_date_context so the save the date web previews would render variables properly from settings.py

### CODE REVIEW

File: guests/save_the_date.py [hunk 1]
```diff
@@ -105,6 +105,10 @@ def get_save_the_date_context(template_id):
         template_id = 'lions-head'
     context = copy(SAVE_THE_DATE_CONTEXT_MAP[template_id])
     context['name'] = template_id
+    context['rsvp_address'] = settings.DEFAULT_WEDDING_REPLY_EMAIL
+    context['site_url'] = settings.WEDDING_WEBSITE_URL
+    context['couple'] = settings.BRIDE_AND_GROOM
+    template_html = render_to_string(SAVE_THE_DATE_TEMPLATE, context=context)
```
Comment (czue) [inline guests/save_the_date.py]: I don't think this line should be here.

**Review by czue — CHANGES_REQUESTED** [review]
Thanks for the contribution! i had one small piece of feedback. If you aren't able to address it I can also make the change later this week.

**Review by dillwishlist — COMMENTED** [review]
Removed offending line.

File: guests/save_the_date.py (hunk 1 above)
Comment (dillwishlist) [reply guests/save_the_date.py]: I removed the last line of the four.  How's that look?

**Review by czue — APPROVED** [review]
perfect, thanks

**Comment by czue** [comment]
thanks for the contribution!

**Pull request merged**